.travis.yml

```diff
@@ -3,7 +3,8 @@ language: python
 services:
   - docker
 
-sudo: false
+os: linux
```

Comment:
Is there a benefit to changing the container?

Reply:
A warning in Travis appeared saying "root: missing os, using the default linux". Sounds like a good idea to specify this explicitly. 
I also specified the distribution to use Bionic (18.04) instead of the default of Xenial (16.04). I don't see any reason not to upgrade, and allows for testing the latest version of Python 3.8.